cmd/kogito/command/shared/install_operator_test.go

```diff
@@ -79,7 +79,7 @@ func Test_InstallOperatorWithYaml(t *testing.T) {
 	crds := &apiextensionsv1beta1.CustomResourceDefinitionList{}
 	err = kubernetes.ResourceC(client).ListWithNamespace(ns, crds)
 	assert.NoError(t, err)
-	assert.Len(t, crds.Items, 7)
+	assert.Len(t, crds.Items, 8)
```

Comment:
We will probably need more tests for trusty operator. I think we need to test more than only that CRDs exists. Would it be possible to test for example if trusty service can be created using CRD? @r00ta  @radtriste What do you think. How are other CRDs tested?